src/views/CardStack.js

```diff
@@ -343,7 +331,16 @@ class CardStack extends Component<DefaultProps, Props, void> {
 
     let panHandlers = null;
 
-    if (this.props.gesturesEnabled) {
+    const cardStackConfig = this.props.router.getScreenConfig(
+      props.navigation,
+      'cardStack'
+    ) || {};
+    const gesturesEnabledConfig = cardStackConfig.gesturesEnabled;
+    const gesturesEnabled =
+      (gesturesEnabledConfig === true || gesturesEnabledConfig === false)
```

Comment:
Okay, done. Also updated the styling of the ternary expression to match what was used elsewhere in the file